gnd/src/main/java/com/google/android/gnd/ui/home/featuredetails/ObservationListFragment.java

```diff
@@ -71,6 +67,7 @@ public View onCreateView(
   @Override
   public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
     super.onViewCreated(view, savedInstanceState);
+    RecyclerView recyclerView = binding.observationListContainer;
```

Comment:
Change from `recyclerView` to `observationList`

Reply:
Done